framework/console/controllers/BaseMigrateController.php

```diff
@@ -639,6 +689,46 @@ protected function getNewMigrations()
     }
 
     /**
+     * Parse the command line migration fields.
+     * @since 2.0.7
+     */
+    protected function parseField()
+    {
+        if ($this->fields === null) {
+            $this->fields = [];
+        }
+
+        foreach ($this->fields as $index => $field) {
+            $chunks = preg_split('/\s?:\s?/', $field, null);
+            $property = array_shift($chunks);
+
+            foreach ($chunks as &$chunk) {
+                if (!preg_match('/(.+?)\(([^)]+)\)/', $chunk)) {
```

Comment:
Maybe the regexp must check for a full match from the beginning to the end? `/^/(.+?)\(([^)]+)\)/$/`